app/src/pages/inside/dashboardItemPage/widgetsGrid/widgetsGrid.jsx

```diff
@@ -105,56 +99,75 @@ export class WidgetsGrid extends PureComponent {
   }
 
   fetchWidgets() {
+    const { userInfo, project } = this.props;
+    const projectRole =
+      userInfo.assigned_projects[project] && userInfo.assigned_projects[project].projectRole;
```

Comment:
If userInfo.assigned_projects[project] will not have projectRole property for some reasons, projectRole constant will be equal to object which contains all info. It i bit strange.